src/styles/index.scss

```diff
@@ -1,3 +1,133 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  font-style: normal;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  margin: 0 auto;
+  padding: 32px 16px 16px;
+
+  &__top {
+    width: 160px;
+    height: 134px;
+    padding: 0 3px;
+  }
+
+  &__bottom {
+    display: flex;
+    flex-direction: column;
+    margin-top: 40px;
+  }
+  &__image-link {
+    display: block;
+    width: 100%;
+  }
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    color: #060b35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__subtitle {
+    color: #616070;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 4px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__review {
+    color: #060b35;
+    text-align: center;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__price-value {
+    color: #060b35;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    color: #fff;
+    text-transform: uppercase;
+    background-color: #00acdc;
+    border-radius: 5px;
+    font-size: 14px;
+    font-weight: 700;
+    text-decoration: none;
+    margin-top: 16px;
+    line-height: 16px;
+  }
+
+  &__buy:hover {
+    background-color: transparent;
+    border: 1px solid #00acdc;
+    color: #00acdc;
+  }
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
```

Comment:
move to a separate file
<img width="586" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/02967cb7-5314-4348-94e7-719cc17936ae">